doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -343,6 +344,8 @@ Matter samples
   * Fixed an issue that prevented nRF Toolbox for iOS in version 5.0.9 from controlling the sample using :ref:`nus_service_readme`.
   * Changed the design of the :ref:`matter_lock_sample_wifi_thread_switching` feature so that support for both Matter over Thread and Matter over Wi-Fi is included in a single firmware image.
 
+* Disabled Read Client functionality in all Matter samples except the :ref:`matter_light_switch_sample` and :ref:`matter_thermostat_sample` samples, and the :ref:`matter_bridge_app` application.
```

Comment:
Minor: I'm not sure about it, but has it ever been enabled? I mean, this is an configuration option that was likely not present in Matter 1.1 and NCS 2.5.0. So informing the user that something that was not enabled before was disabled may not be useful.

Reply:
I'm of two minds. The entry in the protocol section makes it sound like the functionality was always there, but the configuration option to disable it was added. In fact, let me make a new suggestion that incorporates the Kconfig option in the entry to add some clarity.